packages/pwa-kit-runtime/src/utils/ssr-config.server.js

```diff
@@ -79,11 +102,38 @@ export const getConfig = (opts = {}) => {
 
     if (!config) {
         throw new Error(
-            `Application configuration not found!\nPossible configuration file locations:\n${searchPlaces.join(
-                '\n'
-            )}`
+            [
+                'Application configuration not found!',
+                'Possible configuration file locations:',
+                ...searchPlaces
+            ].join('\n')
         )
     }
 
     return config
 }
+
+/**
+ * Gets site specific configurations
+ *
+ * @param {string} [configFile] Optional `configFile` that containing the configution. If it
+ *      isn't specified, the `configFile` will resolved by resolution rules from the `getConfig`
+ * @param {object} opts
+ * @param {string} [opts.buildDirectory] Option path to the folder containing the configution.
+ * @param {string} [opts.siteId] Optional `siteId` that containing the configution. By default,
+ *      it is current site from `app.commerceAPI.parameters.siteId`
+ *
+ * @returns {object} The configution object
+ */
+export function getSiteConfig(configFile = '', {siteId = null, ...rest} = {}) {
+    if (!siteId) {
+        const config = getConfig({...rest})
+
+        siteId = config?.app?.commerceAPI?.parameters?.siteId
```

Comment:
@vinhtrinh the `pwa-kit-runtime` package is agnostic to details like `commerceAPI`. For example when have other projects that use this runtime package that don't use commerce sdks at all. So we can't do something like this.

Reply:
@bendvc , as explained above. Please let me know if you want to remove this helper from the package.